perception/sub8_perception/nodes/buoy_finder.py

```diff
@@ -255,9 +259,10 @@ def image_cb(self, image):
         # Crop out some of the top and bottom to exclude the floor and surface reflections
         height = image.shape[0]
         roi_y = int(0.2 * height)
```

Comment:
They're tuned to the bags I have to remove the worst noise in the bottom and top of frame. I changed them to ros params so they could be easily disabled or tuned if needed now.